### PR TITLE
Most-118: Deploy script for pool

### PR DESCRIPTION
**Pull request by kostekIV**
Just a script which deploys swap

### CODE REVIEW

**Review by Marcin-Radecki — APPROVED** [review]

File: eth/scripts/deploy_stable_swap.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  const LP = await ethers.getContractFactory("StableSwapLP");
+  const lp = await LP.deploy();
+  await lp.waitForDeployment();
+
+  console.log("LP deployed to:", lp.target);
+
+  const Pool = await ethers.getContractFactory("StableSwapTwoPool");
+  const pool = await Pool.deploy();
+  await pool.waitForDeployment();
+  console.log("Pool deployed to:", pool.target);
```
Comment (kostekIV) [inline eth/scripts/deploy_stable_swap.js:36]:
```suggestion
  console.log("Stable Swap deployed to:", pool.target);
```

[the remark this answers is not in the document]
Comment (kostekIV) [reply eth/scripts/deploy_stable_swap.js]: Done

**Review by obrok — APPROVED** [review]

File: eth/scripts/deploy_stable_swap.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+const { ethers } = require("hardhat");
```
[the remark this answers is not in the document]
Comment (kostekIV) [reply eth/scripts/deploy_stable_swap.js:1]: https://cardinal-cryptography.atlassian.net/browse/MOST-129

File: eth/scripts/deploy_stable_swap.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  const tx = await stableSwapFactory.createSwapPair(
+    wrappedToken.target,
+    "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+    1,
+    1,
+    1,
+  );
+  await tx.wait(1);
+
+  console.log(tx);
```
[the remark this answers is not in the document]
Comment (kostekIV) [reply eth/scripts/deploy_stable_swap.js]: todo under Most-129

**Pull request merged**